sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/TableChange.java

```diff
@@ -297,6 +298,24 @@ public int hashCode() {
     }
   }
 
+  /**
+   * Create a TableChange for adding a new Table Constraint
+   */
+  static TableChange addConstraint(Constraint constraint, Boolean validate) {
```

Comment:
Why `Boolean` and not `boolean`? I think `AddConstraint` below actually accepts a primitive, potentially leading a NPE during unboxing.

Reply:
Good catch. 
Was following
```
static TableChange deleteColumn(String[] fieldNames, Boolean ifExists)
```
I think boolean makes more sense here.